pysos.py

```diff
@@ -356,35 +338,18 @@ def __init__(self, path):
         self._observers = []
     
     def __getitem__(self, i):
+        if isinstance(i, slice):
+            return [self[ii] for ii in range(*i.indices(len(self)))]
         key = self._indexes[i]
         return self._dict[key]
-        
-    
-    def get(self, key, default=None):
```

Comment:
Same as previously, no more required because of parent class?

Reply:
Yes, see the unit tests.